app/src/org/commcare/preferences/MainConfigurablePreferences.java

```diff
@@ -101,25 +99,13 @@ protected void setupPrefClickListeners() {
 
         Preference analyticsButton = findPreference(DISABLE_ANALYTICS);
         if (MainConfigurablePreferences.isAnalyticsEnabled()) {
-            analyticsButton.setOnPreferenceClickListener(new Preference.OnPreferenceClickListener() {
-                @Override
-                public boolean onPreferenceClick(Preference preference) {
-                    showAnalyticsOptOutDialog();
-                    return true;
-                }
+            analyticsButton.setOnPreferenceClickListener(preference -> {
+                showAnalyticsOptOutDialog();
+                return true;
             });
         } else {
             getPreferenceScreen().removePreference(analyticsButton);
         }
-
-        Preference printTemplateSetting = findPreference(PREFS_PRINT_DOC_LOCATION);
```

Comment:
made print preference use `FilePreference` construct, so we don't need to handle this explicitly here.